deployment/rdma-cni-daemonset.yaml

```diff
@@ -7,27 +7,28 @@ metadata:
   labels:
     tier: node
     app: rdma-cni
+    name: rdma-cni
 spec:
   selector:
     matchLabels:
       name: rdma-cni
+  updateStrategy:
+    type: RollingUpdate
   template:
     metadata:
       labels:
         tier: node
         app: rdma-cni
         name: rdma-cni
     spec:
-      hostNetwork: true
       nodeSelector:
-        beta.kubernetes.io/arch: amd64
+        kubernetes.io/arch: amd64
```

Comment:
I don't think this is need if we use annotation

Reply:
you mean, if we use docker manifest ?

with docker manifest we will need just one definition of daemonset in the deployment, so ill remove the dedicate ppc64le